packages/web-client/app/components/card-pay/deposit-workflow/supplier-badge/index.hbs

```diff
@@ -1,3 +1,6 @@
-<Boxel::CardContainer>
+<ActionCardContainer
+  @header="Badge"
+  @isComplete={{true}}
```

Comment:
I would go with `@isComplete` like the other `ActionCardContainers` in this PR rather than hardcoding `true`, in case the behaviour/purpose of this card changes.